src/main/java/teammates/storage/sqlentity/FeedbackSession.java

```diff
@@ -86,29 +86,21 @@ protected FeedbackSession() {
         // required by Hibernate
     }
 
-    private FeedbackSession(FeedbackSessionBuilder builder) {
-        this.setName(builder.name);
-        this.setCourse(builder.course);
-
-        this.setCreatorEmail(builder.creatorEmail);
-        this.setInstructions(StringUtils.defaultString(builder.instructions));
-        this.setStartTime(builder.startTime);
-        this.setEndTime(builder.endTime);
-        this.setSessionVisibleFromTime(builder.sessionVisibleFromTime);
-        this.setResultsVisibleFromTime(builder.resultsVisibleFromTime);
-        this.setGracePeriod(Objects.requireNonNullElse(builder.gracePeriod, Duration.ZERO));
-        this.setOpeningEmailEnabled(builder.isOpeningEmailEnabled);
-        this.setClosingEmailEnabled(builder.isClosingEmailEnabled);
-        this.setPublishedEmailEnabled(builder.isPublishedEmailEnabled);
-
-        if (builder.deletedAt != null) {
-            this.setDeletedAt(builder.deletedAt);
-        }
-    }
-
-    @Override
-    public void sanitizeForSaving() {
-        this.instructions = SanitizationHelper.sanitizeForRichText(instructions);
+    public FeedbackSession(String name, Course course, String creatorEmail, String instructions, Instant startTime,
```

Comment:
We might want to keep the builder for this entity. This constructor requires a lot of inputs, and might be difficult to be used in testing e.g I want to test a subset of properties of this entity, but to construct it I have to provide values for everything

Reply:
Would prefer adding this back if required in the future. This is not necessarily required for tests. I can for example have a method that creates a default object, then use the setter to mutate any values I want to test. This would result in a much more readable test as well.

If readability is a concern, and it is a valid point, then this point applies to all entities and not just FeedbackSession. Either way, my point is that we should only implement it when there is a need for it. This is very easy to add later on. Else we can default to a simple constructor. As they always say, YAGNI.